avm/utilities/pipelines/staticValidation/psrule/.ps-rule/min-suppress.Rule.yaml

```diff
@@ -17,6 +17,8 @@ spec:
     - Azure.Automation.ManagedIdentity
     - Azure.Automation.AuditLogs # Diagnostic Settings cannot be set by default, but require user input
     - Azure.Automation.PlatformLogs # Diagnostic Settings cannot be set by default, but require user input
+      # Event Hub specific
+    - Azure.EventHub.Firewall
```

Comment:
@AlexanderSehr @elanzel going forward I'd suggest to add a comment next to the exempted rule with a brief reason for the exeption.